### DISCUSSION TITLE
How to publish to Vercel Templates

### DISCUSSION THREAD

**Discussion by estifanos-neway** [vercel/community#4554] [Help] [answered]
Hello everyone, how can I publish my small app on [Vercel Templates](https://vercel.com/templates)?

**Comment by amyegan** [comment]
Hey! Thanks for your interest in adding a template to our Template Marketplace. We are not accepting new templates at the moment. However, you can create your own Deploy Button to make it easy for people to create their own project based on your repo. 

https://vercel.com/docs/deployments/deploy-button#

**Reply by ghost** [reply]
Are you guys still not accepting new templates?

**Reply by amyegan** [reply]
Not at the moment. But the deploy button will let people create a new project from your repo the same way they would from the templates marketplace.

**Comment by amyegan** [comment, marked as answer]
Good news! We just re-opened Templates Marketplace for new submissions. If you're still interested in sharing your template, please use the [Templates Marketplace submission form](https://vercel.com/templates/submit) to send us the details. 🙂

**Comment by mickasmt** [comment]
Hi @amyegan How can I find out if the project I submitted has been approved or rejected, please? What is the average waiting time after submitting the form?
